src/Core/Notifications/DoNotShowAgainNotificationAction.cs

```diff
@@ -41,6 +41,8 @@ public DoNotShowAgainNotificationAction(IDisabledNotificationsStorage disabledNo
 
         public string CommandText => CoreStrings.Notifications_DontShowAgainAction;
 
+        public bool ShouldDismissAfterAction => true;
```

Comment:
Test Coverage fails because this line is not covered. I do not see any meaningful tests for this.